helper-cli/src/main/kotlin/commands/ListPackagesCommand.kt

```diff
@@ -44,8 +47,7 @@ class ListPackagesCommand : CliktCommand(
     private val matchDetectedLicenses by option(
         "--match-detected-licenses",
         help = "Omit all packages not matching all licenses given by this comma separated list of license identifiers."
-    ).convert { it.split(",").toList() }
-        .default(emptyList())
+    ).convert { File(it.expandTilde()) }.split(",").default(emptyList())
```

Comment:
This should be `.split(",").default(emptyList())`.

Reply:
Done.

Reply:
No, see above, this should *only* be `.split(",").default(emptyList())`.

Reply:
Sorry.